docs/federation/types.md

```diff
@@ -85,6 +85,12 @@ An [ISO 8601][iso8601] date.
 
 Example: `2016-02-19`
 
+## Timezone
+
+A timezone in the form `Area/Location` as used in the [Time Zone Database][tz].
```

Comment:
Is this for display purposes only? I.e. is the timestamp itself still stored as UTC? Otherwise it's too easy to do the wrong thing with the timestamp.

And if it is used only for display purposes then why not use the location for that purpose (to show local time), or the timezone of the computer used by the client displaying this?